.github/workflows/test.yml

```diff
@@ -1,23 +1,30 @@
-name: Build
+name: Node.js Test Runner
+
 on:
+  pull_request:
   push:
     branches:
       - main
 
 jobs:
-  build-and-deploy:
+  build:
+
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x, 18.x]
+
     steps:
-      - name: Checkout 🛎️
-        uses: actions/checkout@v2.3.1
-        with:
-          fetch-depth: 0
+    - uses: actions/checkout@v2
 
-      - name: Install and Build 🔧
-        run: |
-          npm install
-          npm run build
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
 
-      - name: Run Tests 🧪
-        run: |
-          npm run test
+    - run: npm install
+    - run: npm run build --if-present
```

Comment:
Do you need to build?

Reply:
Yup, but we don't need the `--if-present`. We want the CI/CD to rest the typescript builds. Wanting a reason I love not using typescript? No built needed! :P

Fixed in 676c4cd1bbacd250584b55551d7bdb69674978bc.